resources.whatwg.org/build/deploy.sh

```diff
@@ -111,6 +112,32 @@ run_post_build_step "$WEB_ROOT"
 echo "Living standard output to $WEB_ROOT"
 echo ""
 
+header "Starting review drafts..."
+echo "Note: review drafts must be added or changed in a single commit on master"
+for f in "$REVIEW_DRAFTS_DIR"/*.bs; do
+    [ -e "$f" ] || continue # http://mywiki.wooledge.org/BashPitfalls#line-80
+    if [[ "$TRAVIS_PULL_REQUEST" == "true" ]]; then
+        CHANGED_FILES=$(git diff --name-only master..HEAD)
```

Comment:
Since we put the DATE directly in the draft I don't think we need to distinguish between added and changed anymore. Hence we can simply use --name-only instead of --name-status.

I used master..HEAD here so a PR will consider all commits. We'll just have to ensure to land as a single commit on master.

Reply:
Nice!